cmd/lotus-shed/miner-multisig.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/filecoin-project/go-state-types/abi"
 	miner5 "github.com/filecoin-project/specs-actors/v5/actors/builtin/miner"
 
+    miner2 "github.com/filecoin-project/specs-actors/v2/actors/builtin/miner"
```

Comment:
gofmt complains about this:
```suggestion
	miner2 "github.com/filecoin-project/specs-actors/v2/actors/builtin/miner"
```